importer/info.ts

```diff
@@ -17,16 +19,35 @@ const definitionInfo = {
   ],
 };
 
+/**
+ * Framing context
+ */
+const frame = {
+  '@context': {
+    '@vocab': 'https://schema.org/',
+    id: '@id',
+    type: '@type',
+    title: 'http://schema.org/name',
+    description: 'http://schema.org/description',
+  },
+  id: {},
+  type: {},
+  title: {},
+  description: {},
+};
+
 const importInfo = async (importUrl: string): Promise<ParsedResponse> => {
   const result = await fetch(importUrl);
-  const json: Record<string, InfoData> = await result.json();
-  return Object.keys(json).reduce(
-    (acc: ParsedResponse, id) => {
-      acc.info!.push({
-        id,
-        title: json[id].title,
-        description: json[id].description,
-      });
+  const json = await result.json();
+  // TODO: Remove this line once the context has been altered in the jsonld file
```

Comment:
Als dit ticket is geaccepteerd en de @base is uit de context gehaald, kunnen we deze workaround eruit halen.
Zie: https://github.com/amsterdamtimemachine/amsterdam-diaries-data/pull/1